src/System.Collections/tests/Generic/HashSet/HashSet.Generic.Tests.cs

```diff
@@ -335,6 +335,35 @@ public void SetComparer_SequenceEqualTests()
 
         #endregion
 
+        #region TryGetValue
+
+        [Fact]
+        public void HashSet_Generic_TryGetValue_Contains()
+        {
+            T value = CreateT(1);
```

Comment:
I'm not sure how to go about doing that. Would that be done in the file `HashSet.Generic.cs`?